examples/com.sap.vocabularies.UI.v1.DataFieldDefault.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/UI.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.UI.v1" Alias="UI" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Common.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Common.v1" Alias="Common" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema Namespace="datafielddefault.sample" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+      <Annotations Target="my.container/QueryResult/PostingDate">
+        <!-- Based on these annotations, the filter facet for the PostingDate property shall, in addition to the usual date picker,
```

Comment:
I hope I am not completely off but in Filter Facet means the Filter Bar:
Filters are based on the SelectionField annotation which doesn't allow DataFields but only PropertyPath

Reply:
With this pull request, I want to introduce a new type of filter: Besides the usual filters like `PostingDate eq '2020-08-05'`, which you cannot usefully bookmark, you can now set functional filters like `PostingDate eq SAP.GregorianDateFunction(P_DateFunctionID='TODAY')`, which you *can* bookmark.